### PR TITLE
Deadly Runners Welcome Message

### PR DESCRIPTION
**Pull request by Akhito**
Welcome message for the Deadly Runners channel.

### CODE REVIEW

File: deadlywelcome.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+Welcome to the Deadly Runners Division!
+
+This is a display of this clans' best builds, submitted by a wide array of clan members.  Each
```
Comment (crunchyintheory) [inline deadlywelcome.md]: *clan's

File: deadlywelcome.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+Welcome to the Deadly Runners Division!
+
+This is a display of this clans' best builds, submitted by a wide array of clan members.  Each
+build provides a title to each individual.  Rivens are kept seperate from the builds, and 
+screenshots of each build are provided where :riven: is available.  Click :id: to recieve the
+identification of the build via pm.  The order of information on the top of the build goes as
+follows; build name, weapon/warframe name, master holder.
```
Comment (crunchyintheory) [inline deadlywelcome.md]: colon, not semicolon, and it's the creator of that specific build, not the current master holder.

File: deadlywelcome.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+This is a display of this clans' best builds, submitted by a wide array of clan members.  Each
+build provides a title to each individual.  Rivens are kept seperate from the builds, and 
+screenshots of each build are provided where :riven: is available.  Click :id: to recieve the
+identification of the build via pm.  The order of information on the top of the build goes as
+follows; build name, weapon/warframe name, master holder.
+
+--Submission Proccess--
+The specifics on what is allowed for builds is detailed in the deadly runners information,
+please do ``!help deadlyrunners`` to recieve a pm detailing the information.  Submissions
```
Comment (crunchyintheory) [inline deadlywelcome.md]: Incorrect Discord formatting for code escaping, please just use 1 backtick on either side.

File: deadlywelcome.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+are to be private messaged to @Zealous Vindictor#3277 , who will determine if the submitted
```
Comment (crunchyintheory) [inline deadlywelcome.md]: <@156962731084349442> (your ID is in there) is the way mentions are done when a bot sends the message. Remove the space before the comma.

File: deadlywelcome.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
+Welcome to the Deadly Runners Division!
+
+This is a display of this clans' best builds, submitted by a wide array of clan members.  Each
+build provides a title to each individual.  Rivens are kept seperate from the builds, and 
+screenshots of each build are provided where :riven: is available.  Click :id: to recieve the
+identification of the build via pm.  The order of information on the top of the build goes as
+follows; build name, weapon/warframe name, master holder.
+
+--Submission Proccess--
```
Comment (crunchyintheory) [inline deadlywelcome.md]: Bold this

File: deadlywelcome.md [hunk 6]
```diff
@@ -1,14 +1,14 @@
...
 
-This is a display of this clans' best builds, submitted by a wide array of clan members.  Each
+This is a display of this clan's best builds, submitted by a wide array of clan members.  Each
 build provides a title to each individual.  Rivens are kept seperate from the builds, and 
 screenshots of each build are provided where :riven: is available.  Click :id: to recieve the
 identification of the build via pm.  The order of information on the top of the build goes as
 follows; build name, weapon/warframe name, master holder.
 
---Submission Proccess--
+*Submission Proccess*
```
Comment (crunchyintheory) [inline deadlywelcome.md]: That's italics, not bold

**Comment by Akhito** [comment]
Reworked deadlyrunners.md and deadlywelcome.md to align with requested alterations.

File: deadlywelcome.md (hunk 6 above)
Comment (crunchyintheory) [reply deadlywelcome.md]: Also, *Process

**Comment by crunchyintheory** [comment]
Looking good! Please update the button/reaction descriptions to be consistent with the format used in 94132c804acb73fdb4872792891aae53674fe310 and I can merge.

**Pull request merged**